### PR TITLE
Better update channel selector design

### PR DESCRIPTION
**Pull request by Jerome-Herbinet**
## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by jancborchardt — COMMENTED** [review]
Looks nice! But only the right part (the channel itself) should be inside the border, no?

**Comment by Jerome-Herbinet** [comment]
> Looks nice! But only the right part (the channel itself) should be inside the border, no?

@jancborchardt it's done and looking fine IMO : 

If it's OK for you, tell me, so I can squash all this.

File: apps/updatenotification/src/components/UpdateNotification.vue [hunk 1]
```diff
@@ -509,6 +506,9 @@ export default {
 			cursor: pointer;
 			margin-left: 3px;
 			display: inline-block;
+			padding: 10px;
+			border-radius: 10px;
+			border: 2px solid var(--color-border-dark);
```
Comment (szaimen) [inline apps/updatenotification/src/components/UpdateNotification.vue:511]:
```suggestion
			border: 2px solid var(--color-border-dark);
```

**Comment by Jerome-Herbinet** [comment]
@szaimen I let you check if everything is OK for you.

**Review by szaimen — APPROVED** [review]
Tested and seems to look and work fine

**Comment by susnux** [comment]
@Jerome-Herbinet can you resolve the conflicts?

**Review by susnux — APPROVED** [review]

**Comment by Jerome-Herbinet** [comment]
See respawn https://github.com/nextcloud/server/pull/39927

**Pull request closed without merging**